cmd/gsw_service.go

```diff
@@ -17,6 +17,9 @@ import (
 	"go.uber.org/zap"
 
 	"github.com/AarC10/GSW-V2/proc"
+
+	"net/http"
+	_ "net/http/pprof"
```

Comment:
?

Reply:
- net/http to start the server
- `_ "net/http/pprof" to bring in the pprof package. _ to keep it anonymous or else it would be an unused import. need to import it bc the init() in that package sets it up to listen on the global httpserver